src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -86,19 +140,49 @@
  * the {@code aliasList}:
  * </p>
  * <pre>
- * &#64;SuppressWarnings("paramnum")
- * public void needsLotsOfParameters(@SuppressWarnings("unused") int a,
- *   int b, int c, int d, int e, int f, int g, int h) {
- *   ...
+ * &lt;module name=&quot;TreeWalker&quot;&gt;
+ *   &lt;module name=&quot;ParameterNumber&quot;/&gt;
+ *   &lt;module name=&quot;SuppressWarningsHolder&quot;&gt;
+ *     &lt;property name=&quot;aliasList&quot; value=
+ *       &quot;com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramnum&quot;/&gt;
+ *   &lt;/module&gt;
+ * &lt;/module&gt;
+ * &lt;module name=&quot;SuppressWarningsFilter&quot;/&gt;
+ * </pre>
+ * <pre>
+ * class Test {
+ *
+ *     public void needsLotsOfParameters int a, // violation
+ *       int b, int c, int d, int e, int f, int g, int h) {
+ *       // ...
+ *     }
+ *
+ *     &#64;SuppressWarnings("paramnum")
+ *     public void needsLotsOfParameters1 int a, // violation suppressed
+ *       int b, int c, int d, int e, int f, int g, int h) {
+ *       //  ...
+ *     }
+ *
  * }
  * </pre>
  * <p>
  * It is possible to suppress all the checkstyle warnings with the argument {@code "all"}:
  * </p>
  * <pre>
- * &#64;SuppressWarnings("all")
- * public void someFunctionWithInvalidStyle() {
- *   //...
+ * &lt;module name=&quot;TreeWalker&quot;&gt;
+ *   &lt;module name=&quot;MemberName&quot;/&gt;
+ *   &lt;module name=&quot;NoWhitespaceAfter&quot;/&gt;
+ *   &lt;module name=&quot;SuppressWarningsHolder&quot;/&gt;
+ * &lt;/module&gt;
+ * &lt;module name=&quot;SuppressWarningsFilter&quot;/&gt;
+ * </pre>
+ * <pre>
+ * class Test {
+ *
+ *    private int [] ARR; // 2 violations
+ *    &#64;SuppressWarnings("all")
+ *    private int [] ARRAY; // 2 violations suppressed
```

Comment:
as config of holder is default, lets merge this example to default confg example.

Reply:
done

Reply:
Remove this example.

It is covered by default example

Reply:
done